src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Endpoints/Completion.cs

```diff
@@ -271,4 +282,135 @@ private static TextEdit BuildRevertedEdit(TextEdit provisionalTextEdit)
         var formattingOptions = _formattingOptionsProvider.GetOptions(document.TextDocumentIdentifier.Uri);
         return Task.FromResult(formattingOptions);
     }
+
+    [JsonRpcMethod(LanguageServerConstants.RazorSnippetCompletionEndpointName, UseSingleObjectParameterDeserialization = true)]
+    public CompletionItem[]? GetSnippetCompletions(RazorSnippetCompletionParams completionParams, CancellationToken _)
+    {
+        if (completionParams.TextSpan.Length == 0)
```

Comment:
So I a user invokes completion with `Ctrl+Space`, then starts typing, they won't get snippets? Is that expected behaviour?